tensorflow_lite_support/python/task/text/nl_classifier.py

```diff
@@ -0,0 +1,101 @@
+# Copyright 2022 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""NL Classifier task."""
+
+import dataclasses
+
+from tensorflow_lite_support.python.task.core import base_options as base_options_module
+from tensorflow_lite_support.python.task.processor.proto import classifications_pb2
+from tensorflow_lite_support.python.task.processor.proto import nl_classification_options_pb2
+from tensorflow_lite_support.python.task.text.pybinds import _pywrap_nl_classifier
+
+_CppNLClassifier = _pywrap_nl_classifier.NLClassifier
+_BaseOptions = base_options_module.BaseOptions
+_NLClassificationOptions = nl_classification_options_pb2.NLClassificationOptions
+
+
+@dataclasses.dataclass
+class NLClassifierOptions:
+  """Options for the NL classifier task."""
+  base_options: _BaseOptions
+  nl_classification_options: _NLClassificationOptions = _NLClassificationOptions()
```

Comment:
@kinaryml We got feedback from internal review that the Python NLClassifierOptions shouldn't expose these customization options that exist in C++ because it is meant to be for advanced users only. So please remove this proto and use the default options.

Instead, this should be replaced by the [ClassificationOptions](https://github.com/tensorflow/tflite-support/blob/master/tensorflow_lite_support/python/task/processor/proto/classification_options_pb2.py). These filtering options aren't available in the C++ layer so you should implement them in the Python layer.

Reply:
@khanhlvg I've added the filtering options in the Python layer along with some unit tests and removed any customization options associated with `NLClassificationOptions`. PTAL.